test/MerkleLib.Claims.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from "chai";
+import { merkleLibFixture } from "./MerkleLib.Fixture";
+import { Contract, BigNumber } from "ethers";
+
+describe("MerkleLib Claims", async function () {
+  let merkleLibTest: Contract;
```

Comment:
nit: in the other test files we've done the declaration of these types of variables outside of the `describe` block.

Reply:
Okay, sounds good.